src/function/table/hnsw/query_hnsw_index.cpp

```diff
@@ -6,12 +6,16 @@
 #include "common/types/value/nested.h"
 #include "function/table/bind_data.h"
 #include "function/table/hnsw/hnsw_index_functions.h"
+#include "planner/operator/logical_hash_join.h"
+#include "planner/operator/logical_table_function_call.h"
+#include "planner/planner.h"
 #include "processor/execution_context.h"
 #include "storage/index/hnsw_index.h"
 #include "storage/index/hnsw_index_utils.h"
 #include "storage/index/index_utils.h"
 #include "storage/storage_manager.h"
 #include "storage/store/rel_table.h"
+#include <binder/query/reading_clause/bound_table_function_call.h>
```

Comment:
why is this include surrounded by `<>`

Reply:
hmm fixed it. accidentally completed by IDE.